pkg/util/clouderrors/errors.go

```diff
@@ -0,0 +1,31 @@
+package clouderrors
```

Comment:
Let's just put this into pkg/api/validate rather than starting a new package.  It's dealing with error translation to CloudErrors which is a pretty pkg/api/validate-specific thing (at least now).